backend/src/main/java/ca/bc/gov/hlth/mohums/model/BulkRemovalRequest.java

```diff
@@ -0,0 +1,24 @@
+package ca.bc.gov.hlth.mohums.model;
+
+import java.util.List;
+import java.util.Map;
+
+public class BulkRemovalRequest {
+
+    private Map<String, List<Object>> userRolesForRemoval;
```

Comment:
I have not tried putting String in there, although I'm not sure how parsing would work in that case.

UMS parses this Object to a LinkedHashMap. It represents Keycloak `[RoleRepresentation](https://www.keycloak.org/docs-api/22.0.1/rest-api/index.html#RoleRepresentation)` class. I did not create a model for this class, because that's how we handle things across UMS as a whole - passing Objects to and from Keycloak API. This makes development less clear, but technically, we do not have to make any updates if model changes on the Keycloak side of things - just proxying what Keycloak sends.